### PR TITLE
[IMP] Product weight on receipt

### PR DESCRIPTION
**Pull request by nicolasj77**
Ajout du poids sur le bon d'expédition et le PDF

### CODE REVIEW

File: stock_product_weight_on_receipt/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class Product(models.Model):
```
Comment (remytms) [inline stock_product_weight_on_receipt/models/stock.py]: Change class name to `StockPackOperation`.

File: stock_product_weight_on_receipt/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs
+# Nicolas Jamoulle, <nicolas@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Weight product on receipt",
+    "version": "1.0",
```
Comment (remytms) [inline stock_product_weight_on_receipt/__openerp__.py]: Put version : `9.0.1.0`.

File: stock_product_weight_on_receipt/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (remytms) [inline stock_product_weight_on_receipt/__init__.py]: EOL

File: stock_product_weight_on_receipt/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+#. module: stock_product_weight_on_receipt
+#: model:ir.model.fields,field_description:stock_product_weight_on_receipt.field_stock_pack_operation_display_weight
+msgid "Weight"
+msgstr "Poids"
+
+#. module: stock_product_weight_on_receipt
+#: model:ir.model.fields,field_description:stock_product_weight_on_receipt.field_stock_pack_operation_display_unit
+msgid "Weight Unit"
+msgstr "Unité de poids"
```
Comment (remytms) [inline stock_product_weight_on_receipt/i18n/fr.po]: EOL

File: stock_product_weight_on_receipt/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+from . import stock
```
Comment (remytms) [inline stock_product_weight_on_receipt/models/__init__.py]: EOL

File: stock_product_weight_on_receipt/views/stock_view.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='pack_operation_product_ids']/tree/field[@name='fresh_record']"
+                       position="before">
+                    <field name="display_weight"/>
+                    <field name="display_unit"/>
+                </xpath>
+            </field>
+        </record>
+    </data>
+</openerp>
```
Comment (remytms) [inline stock_product_weight_on_receipt/views/stock_view.xml]: EOL

**Review by remytms — CHANGES_REQUESTED** [review]
Good ! :) Some changes.

File: stock_product_weight_on_receipt/models/stock.py (hunk 1 above)
Comment (nicolasj77) [reply stock_product_weight_on_receipt/models/stock.py]: Done

File: stock_product_weight_on_receipt/__openerp__.py (hunk 2 above)
Comment (nicolasj77) [reply stock_product_weight_on_receipt/__openerp__.py]: Done

File: stock_product_weight_on_receipt/__init__.py (hunk 3 above)
Comment (nicolasj77) [reply stock_product_weight_on_receipt/__init__.py]: Done

File: stock_product_weight_on_receipt/i18n/fr.po (hunk 4 above)
Comment (nicolasj77) [reply stock_product_weight_on_receipt/i18n/fr.po]: Done

File: stock_product_weight_on_receipt/models/__init__.py (hunk 5 above)
Comment (nicolasj77) [reply stock_product_weight_on_receipt/models/__init__.py]: Done

File: stock_product_weight_on_receipt/views/stock_view.xml (hunk 6 above)
Comment (nicolasj77) [reply stock_product_weight_on_receipt/views/stock_view.xml]: Done

**Comment by nicolasj77** [comment]
Fix review

File: stock_product_weight_on_receipt/i18n/fr.po [hunk 7]
```diff
@@ -0,0 +1,41 @@
+# Translation of Odoo Server.
```
Comment (remytms) [inline stock_product_weight_on_receipt/i18n/fr.po:1]: Re-generate this file to match our module. :)

**Review by remytms — CHANGES_REQUESTED** [review]
Just translation file.

File: stock_product_weight_on_receipt/i18n/fr.po (hunk 7 above)
Comment (remytms) [reply stock_product_weight_on_receipt/i18n/fr.po:1]: Oh, it's ok. Sorry.

**Review by remytms — APPROVED** [review]

**Pull request merged**